Source/BusinessMomentsAdaptorImage/index.ts

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+declare var process: {
+    env: {
+        WH_TOKEN: string;
+    };
+};
+
+// Initialize express and define a port
+const app = express();
+const PORT = 3008;
+
+// Tell express to use body-parser's JSON parsing
+
+app.use(bodyParser.json());
+
+app.post('/api/webhooks-ingestor', (req: any, res: any) => {
+    console.log(req.headers.authorization);
+    if (req.headers.authorization.split(' ')[1] === process.env.WH_TOKEN) {
```

Comment:
If no auth header is provided in the request it will cause an exception, because the split method (req.headers.authorization.split) will return an empty array.

Reply:
I suggest you drop the split and expect the type of auth header to be provided as environment variable `Basic bTM6am9obmNhcm1hY2s=`. This will also make the code cleaner.

Reply:
I have changed the env variable to "Basic bTM6am9obmNhcm1hY2s=". but the type in ts code is a string. Do we have a type representing the env variable?